### PR TITLE
inventec_d7054q28b:Add new platform

### PR DESCRIPTION
**Pull request by pjcheninventec**
Product highlights for Inventec D7054q28B are as follows:

Performance
	•2678M Packets per Second*
	•1.8 Terabits per Second Throughput
	•Line Rate L2/L3 Forwarding
	•16MB Packet Buffer
	•48x25G SFP28+6x100G QSFP28
Scalability
	•136K MAC Entries (HW Capable)
	•84K IPv4 Host Routes
	•160K IPv4 Routes
	•84K IPv6/64 Routes
	•20K IPv6/128 Routes
	•64K Mroutes
	•32K IPv6 Mroutes
	•4K VLANs
High Availability
	•1 + 1 Hot-Swappable & Redundant Power Supply
	•2 x SPI Flash Supports Boot Recovery
	•3 + 1 Hot-Swappable & Redundant Fans
	•802.3ad Link Aggregation/LACP
	•256 Ports/Channel
	• 1024 Groups per System
Control Plane
	•CPU Options
	•2.0 GHz x86 Octal-Core 8GB to 16GB DDR4
	•2.2 GHz x86 Quad-Core 8GB DDR4
	•1.6 GHz x86 Quad-Core 8GB DDR4
	•16MB SPI Boot Flash with backup
Flexible Storage
	•8-128GB SSD for Mass Storage
	•1 x USB Port for External Storage
Software Support
	•Inventec IN-OS
	•ONIE
	•Open Source Software Provided as RPM
	•Chef and Puppet Client Integration
	•Bash Shell

Signed-off-by: pj.chen <chen.pj@inventec.com>

### CODE REVIEW

File: machine/inventec/inventec_d7054q28b/machine.make [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+endif
+
+# Vendor ID -- IANA Private Enterprise Number:
+# http://www.iana.org/assignments/enterprise-numbers
+# Inventec Corporation
+VENDOR_ID = 6569
+
+# Skip the i2ctools and the onie-syseeprom command for this platform
+I2CTOOLS_ENABLE = yes
+I2CTOOLS_SYSEEPROM = no
```
Comment (cbrune) [inline machine/inventec/inventec_d7054q28b/machine.make]: It is no longer necessary to specify `I2CTOOLS_SYSEEPROM = no`, as that is the default.

File: machine/inventec/inventec_d7054q28b/rootconf/sysroot-init/networking.sh [hunk 2]
```diff
@@ -0,0 +1,182 @@
+#!/bin/sh
```
Comment (cbrune) [inline machine/inventec/inventec_d7054q28b/rootconf/sysroot-init/networking.sh]: why is the file needed ?

it is a very old version of the ONIE core /etc/init.d/networking.sh

several bugs have been fixed in the core, which are not reflected here.

i would get rid of this and use the core version

**Comment by cbrune** [comment]
This looks pretty good for the most part.  Just a couple of things to straighten out.

**Comment by pjcheninventec** [comment]
I'm very appreciated for your feedback.  I modify the patch as your suggestion, hope it would fit and thanks for your time.

**Comment by cbrune** [comment]
Very nice, thanks for contributing the update.

Pushed to master:
- 682e39a inventec_d7054q28b:Add new platform

**Pull request closed without merging**